homeassistant/components/ring/event.py

```diff
@@ -0,0 +1,121 @@
+"""Component providing support for ring events."""
+
+from dataclasses import dataclass
+from typing import Generic
+
+from ring_doorbell import RingCapability, RingEvent as RingAlert
+from ring_doorbell.const import KIND_DING, KIND_INTERCOM_UNLOCK, KIND_MOTION
+
+from homeassistant.components.event import (
+    EventDeviceClass,
+    EventEntity,
+    EventEntityDescription,
+)
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import RingConfigEntry
+from .coordinator import RingListenCoordinator
+from .entity import RingBaseEntity, RingDeviceT
+
+
+@dataclass(frozen=True, kw_only=True)
+class RingEventEntityDescription(EventEntityDescription, Generic[RingDeviceT]):
+    """Base class for event entity description."""
+
+    capability: RingCapability
+
+
+EVENT_DESCRIPTIONS: tuple[RingEventEntityDescription, ...] = (
+    RingEventEntityDescription(
+        key=KIND_DING,
+        translation_key=KIND_DING,
+        device_class=EventDeviceClass.DOORBELL,
+        event_types=[KIND_DING],
+        entity_registry_enabled_default=True,
+        capability=RingCapability.DING,
+    ),
+    RingEventEntityDescription(
+        key=KIND_MOTION,
+        translation_key=KIND_MOTION,
+        device_class=EventDeviceClass.MOTION,
+        event_types=[KIND_MOTION],
+        entity_registry_enabled_default=True,
+        capability=RingCapability.MOTION_DETECTION,
+    ),
+    RingEventEntityDescription(
+        key=KIND_INTERCOM_UNLOCK,
+        translation_key=KIND_INTERCOM_UNLOCK,
+        device_class=EventDeviceClass.BUTTON,
+        event_types=[KIND_INTERCOM_UNLOCK],
+        entity_registry_enabled_default=True,
```

Comment:
the enabled_default to True can be removed

Reply:
Removed